### DISCUSSION TITLE
Youtube Video Project

### DISCUSSION THREAD

**Discussion by AsimKhan2019** [TheOdinProject/curriculum#24119] [General]
Hi, 

How can I get started on the YouTube Video Project and where can I find the template to follow and replicate? 

Regards, 

Asim Khan

**Comment by ChargrilledChook** [comment]
Hello @AsimKhan2019, this project was archived when we re-wrote our HTML / CSS content last year and is no longer available.

If this has come via Microverse, you should know we are not affiliated with them in any way and they are using our content without permission. We can't provide support for anything Microverse related, although you are welcome to come and do our curriculum for free. Feel free to [visit our site here](https://www.theodinproject.com/).

Hope that clears things up, all the best.
